<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpringAngles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CadGraphicsView" name="gvMain">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actFileNew"/>
    <addaction name="actFileOpen"/>
    <addaction name="actFileSave"/>
    <addaction name="actFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actFileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="actToolSelectObject"/>
    <addaction name="actToolAddFixedPoint"/>
    <addaction name="actToolAddSpring"/>
    <addaction name="actToolRemoveObject"/>
   </widget>
   <widget class="QMenu" name="menu_Zoom">
    <property name="title">
     <string>&amp;Zoom</string>
    </property>
    <addaction name="actZoomIn"/>
    <addaction name="actZoomOut"/>
    <addaction name="actZoomAll"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tool"/>
   <addaction name="menu_Zoom"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actToolSelectObject"/>
   <addaction name="actToolAddFixedPoint"/>
   <addaction name="actToolAddSpring"/>
   <addaction name="actToolRemoveObject"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomAll"/>
   <addaction name="separator"/>
   <addaction name="actAdjust"/>
  </widget>
  <action name="actFileNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actFileOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actFileSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actFileSaveAs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actFileExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actToolAddFixedPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
   <property name="text">
    <string>Add fixed point</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actToolAddSpring">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InsertLink"/>
   </property>
   <property name="text">
    <string>Add spring</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actToolRemoveObject">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Remove object</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actToolSelectObject">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
   </property>
   <property name="text">
    <string>Select object</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
   </property>
   <property name="text">
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actZoomAll">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomFitBest"/>
   </property>
   <property name="text">
    <string>Zoom &amp;all</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actAdjust">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Adjust</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CadGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>CadGraphicsView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
